services/datamanager/builtin/builtin.go

```diff
@@ -193,16 +209,15 @@ func (svc *builtIn) flushCollectors() {
 
 // Parameters stored for each collector.
 type collectorAndConfig struct {
-	Collector  data.Collector
-	Attributes dataCaptureConfig
+	Collector data.Collector
+	Config    dataCaptureConfig
 }
 
 // Identifier for a particular collector: component name, component model, component type,
 // method parameters, and method name.
 type componentMethodMetadata struct {
 	ComponentName  string
 	ComponentModel resource.Model
```

Comment:
[q]: Why was MethodParams removed from here?

Reply:
I removed it because `componentMethodMetadata` is basically uniquely defining/identifying "thing that can be collected", while `dataCaptureConfig` defines "configuration for a specific collector". I _thought_ MethodParams doesn't need to be in the former because we don't support capturing the same component/method with multiple collectors at the same time with different params... but looking in the UI, we totally do support that (e.g. capturing a camera as png and jpeg). So readded